packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -36,9 +36,12 @@ import { AppRouter, FlatRoutes } from '@backstage/core-app-api';
 import { CatalogGraphPage } from '@backstage/plugin-catalog-graph';
 import { RequirePermission } from '@backstage/plugin-permission-react';
 import { catalogEntityCreatePermission } from '@backstage/plugin-catalog-common/alpha';
+import { NotificationsPage } from '@backstage/plugin-notifications';
+import * as plugins from './plugins';
 
 const app = createApp({
   apis,
+  plugins: Object.values(plugins),
```

Comment:
Why do we need this? Generally we have moved away from needing to do this.

Reply:
@awanlin Can you please suggest proper replacement?

Reply:
Any updates on this?

Reply:
Not really, the 3 items I highlighted in my review comment have not been addressed yet which I see as blockers from this moving forward. 

For this specific comment I don't know an alternative, in most plugins it just works without needing this. Not sure why that isn't the case for the Notifications plugin.

@Rugvip any thoughts on this?

Reply:
The signals FE plugin is not discovered otherwise. It has been long time since I was looking into it. IIRC, the signals plugin has no component to be attached into the tree. But I might be wrong now.

Reply:
> Not really, the 3 items I highlighted in my review comment have not been addressed yet which I see as blockers from this moving forward.

This effort is not abandoned, just postponed due to recent higher-priority tasks. I will come back to it later, will start with the documentation.

Reply:
This has been resolved with the addition of `<SignalsDisplay />`